LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016, Jim Crist
+Copyright (c) 2016-2022, Jim Crist. 2023, Dask Developers.
```

Comment:
Updating last name. Also, AFAIK you should only have to put the start year on the license.

```suggestion
Copyright (c) 2016, Jim Crist-Harif. 2023, Dask Developers.
```

Reply:
Of course. A lots changed since 2016 : )